tests/unit/test_config.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import asdict
+
+import pytest
+
+from sutta_publisher.shared.config import Config
+
+
+@pytest.mark.vcr
+def test_config_for_publication(scpub1_data):
```

Comment:
Names of tests should follow the convention `test_should_...` so for example 
```suggestion
def test_should_create_config_file():
```